src/styles/card.scss

```
@@ -0,0 +1,75 @@
+@mixin flexed {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.item {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  width: $card-width;
+  flex-direction: column;
+  align-items: center;
+  padding: 30px 14px 15px;
+  max-height: 403px;
+  font-style: normal;
+  font-size: 11px;
+  line-height: 13px;
+  margin: 0 24px 50px;
+
+}
+
+.item__img {
+  width: 160px;
+  height: 134px;
+}
+
+.item__name {
+  margin: 45px 0 0;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
```

```suggestion
    &__img {
      width: 160px;
      height: 134px;
  }

  &__name {
    margin: 45px 0 0;
    font-weight: 500;
    font-size: 12px;
    line-height: 15px;
  }
}
```
Используй scss вложенность во всех файлах